client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -19,25 +19,43 @@ const CancelPurchaseForm = React.createClass( {
 		surveyStep: React.PropTypes.number.isRequired,
 		showSurvey: React.PropTypes.bool.isRequired,
 		defaultContent: React.PropTypes.node.isRequired,
-		onInputChange: React.PropTypes.func.isRequired
+		onInputChange: React.PropTypes.func.isRequired,
+		isJetpack: React.PropTypes.bool.isRequired
 	},
 
 	getInitialState() {
 		// shuffle reason order, but keep anotherReasonOne last
-		const questionOneOrder = shuffle( [
+		var questionOneOrder = shuffle( [
```

Comment:
Using `var` is discouraged in favor of `let` and `const` - since these can change, you can use `let`.